web/src/containers/Routes/index.tsx

```diff
@@ -11,10 +11,28 @@ import { withScrollRestoration } from "../../utility/withScrollRestoration";
 import { ReactNode } from "react";
 import { useRecoilValue } from "recoil";
 
+const showGemsOnly = false;
+
+function FilterGemsOnly(taskItems: TaskListItem[]) {
+  const newTasksItems = [];
+  for (let i = 0; i < taskItems.length; i++) {
+    const nextStep: TaskListItem | undefined = taskItems[i+1];
+
+    if (taskItems[i].type === "fragment_Step" && nextStep?.type !== "gem_step") {
```

Comment:
should probably be fragment_step (it's capitalized right now). same comment for line 45 and in TaskList/index.ts

Reply:
Fixed too